Dockerfile

```diff
@@ -6,10 +6,11 @@ COPY . /opt/sapcli
 
 RUN cd /opt/sapcli &&\
 	python3 -m pip install -r requirements.txt &&\
-	chmod +x /opt/sapcli/sapcli &&\
-	echo '#!/bin/sh' > /usr/bin/sapcli &&\
-	echo 'python3  /opt/sapcli/sapcli "$@"' >> /usr/bin/sapcli &&\
-	chmod +x /usr/bin/sapcli
+	dos2unix /opt/sapcli/bin/sapcli &&\
+	chmod +x /opt/sapcli/bin/sapcli
```

Comment:
Why? The file is already marked executable.
```
$ ls -al bin/sapcli 
-rwxr-xr-x 1 jfilak jfilak 5680 Jul 20 21:13 bin/sapcli
```

Reply:
Do you have to do this because you work on Windows?

Reply:
I work on both Windows and Linux; I did quickly check this and it seems that the executable flag still survives even if you are running the docker build on Windows, I guess I added this command out of habit - I will remove this line